### ISSUE TITLE
Dot-style QR Codes not scanning in Flutter

### ISSUE THREAD

**Issue by ericaliebmann** [juliansteenbakker/mobile_scanner#900]
Using Mobile Scanner 3.5.5 on flutter IOS

The attached QR code scans fine on iOS camera, but not through mobile scanner
<img width="259" alt="Screenshot 2023-12-17 at 8 55 07 PM" src="https://github.com/juliansteenbakker/mobile_scanner/assets/79177074/2fba06fa-b78a-4ec6-99b0-d91b9d3a72b0">

**Comment by ericaliebmann** [comment]
@juliansteenbakker we can put a $300 bounty for this fix!

**Comment by juliansteenbakker** [comment]
Do you have a source where i can generate these kind of QR codes? The example attached is also slightly warped which may impact scanning performance

**Comment by ericaliebmann** [comment]
I don't have a way to generate these, but we can try to find one for you.
We've found a few QR codes like this out in the wild. We can get you a
better picture

**Comment by ericaliebmann** [comment]
<img width="470" alt="Screenshot 2023-12-19 at 9 51 42 AM" src="https://github.com/juliansteenbakker/mobile_scanner/assets/79177074/a0f632a0-b24a-4e08-ac5f-d6a8cd4ce01f">

**Comment by SputNikPlop** [comment]
@juliansteenbakker I found the best way to test this is to use Chrome's function for qr codes or I used https://www.qrcode-monkey.com/ and customized the design

**Comment by SputNikPlop** [comment]
@ericaliebmann I may have time to take a look at this soon. Have you found any other qr codes like this?

**Comment by ericaliebmann** [comment]
No, this is the only one we’ve found so far. Do you still need a better
picture? I can work it that

**Comment by juliansteenbakker** [comment]
There is nothing much we can do. The underlying MLKit framework does not recognize this type of code.
